### PR TITLE
Added new function for clean confirmable message list from coap library.

### PR DESCRIPTION
**Pull request by juhhei01**

### CODE REVIEW

**Comment by juhhei01** [comment]
@AnttiKauppila Review this please.

File: nsdl-c/sn_nsdl_lib.h [hunk 1]
```diff
@@ -697,6 +697,16 @@ extern int8_t sn_nsdl_set_block_size(struct nsdl_s *handle, uint16_t block_size)
  */
 extern int8_t sn_nsdl_set_duplicate_buffer_size(struct nsdl_s *handle, uint8_t message_count);
 
+/**
+ * \fn int8_t sn_nsdl_clear_confirmable_list(struct nsdl_s *handle)
```
Comment (terhei) [inline nsdl-c/sn_nsdl_lib.h]: rename this -> clear_coap_resending_queue

**Comment by juhhei01** [comment]
renamed already done

**Comment by terhei** [comment]
awesome.

**Pull request merged**